<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stationParametersDialog</class>
 <widget class="QDialog" name="stationParametersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>select parameters you want to add:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_load">
       <property name="toolTip">
        <string>load predefined parameters</string>
       </property>
       <property name="statusTip">
        <string>load predefined parameters</string>
       </property>
       <property name="text">
        <string> load</string>
       </property>
       <property name="icon">
        <iconset resource="myresources.qrc">
         <normaloff>:/icons/icons/folder-open-5.png</normaloff>:/icons/icons/folder-open-5.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="toolTip">
        <string>save this parameter for further use</string>
       </property>
       <property name="statusTip">
        <string>save this parameter for further use</string>
       </property>
       <property name="text">
        <string> save</string>
       </property>
       <property name="icon">
        <iconset resource="myresources.qrc">
         <normaloff>:/icons/icons/document-export.png</normaloff>:/icons/icons/document-export.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_available">
         <property name="toolTip">
          <string>availability of this station for scans</string>
         </property>
         <property name="statusTip">
          <string>availability of this station for scans</string>
         </property>
         <property name="title">
          <string>available</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QRadioButton" name="radioButton_available_yes">
            <property name="toolTip">
             <string>station can be scheduled</string>
            </property>
            <property name="statusTip">
             <string>station can be scheduled</string>
            </property>
            <property name="text">
             <string>yes</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_available_no">
            <property name="toolTip">
             <string>station cannot be scheduled</string>
            </property>
            <property name="statusTip">
             <string>station cannot be scheduled</string>
            </property>
            <property name="text">
             <string>no</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_availableForFillinmode">
         <property name="toolTip">
          <string>availability of this station for fillinmode scans</string>
         </property>
         <property name="statusTip">
          <string>availability of this station for fillinmode scans</string>
         </property>
         <property name="title">
          <string>available for fillin mode</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QRadioButton" name="radioButton_availableFillin_yes">
            <property name="toolTip">
             <string>station can be scheduled for fillin mode scans</string>
            </property>
            <property name="statusTip">
             <string>station can be scheduled for fillin mode scans</string>
            </property>
            <property name="text">
             <string>yes</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_availableFillin_no">
            <property name="toolTip">
             <string>station cannot be scheduled for fillin mode scans</string>
            </property>
            <property name="text">
             <string>no</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_tagalong">
         <property name="title">
          <string>tagalong</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QRadioButton" name="radioButton_tagalong_yes">
            <property name="toolTip">
             <string>use tagalong mode for this station</string>
            </property>
            <property name="statusTip">
             <string>use tagalong mode for this station</string>
            </property>
            <property name="text">
             <string>yes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_tagalong_no">
            <property name="toolTip">
             <string>do not use tagalong mode for this station</string>
            </property>
            <property name="statusTip">
             <string>do not use tagalong mode for this station</string>
            </property>
            <property name="text">
             <string>no</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>general</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox_weight">
            <property name="text">
             <string>weight</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_weight">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>weight of this station for selection of best scan</string>
            </property>
            <property name="statusTip">
             <string>weight of this station for selection of best scan</string>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox_maxSlewTime">
            <property name="text">
             <string>max slew time</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBox_maxSlewTime">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>maximum allowed slew time to next source in seconds</string>
            </property>
            <property name="statusTip">
             <string>maximum allowed slew time to next source in seconds</string>
            </property>
            <property name="suffix">
             <string> [s]</string>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="singleStep">
             <number>60</number>
            </property>
            <property name="value">
             <number>600</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="checkBox_maxWaitTime">
            <property name="text">
             <string>max wait time</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="spinBox_maxWaitTime">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>maximum allowed wait time for slower stations in seconds</string>
            </property>
            <property name="statusTip">
             <string>maximum allowed wait time for slower stations in seconds</string>
            </property>
            <property name="suffix">
             <string> [s]</string>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="singleStep">
             <number>60</number>
            </property>
            <property name="value">
             <number>600</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="checkBox_minElevation">
            <property name="text">
             <string>min elevation</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_minElevation">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>minimum elevation for observations in degrees</string>
            </property>
            <property name="statusTip">
             <string>minimum elevation for observations in degrees</string>
            </property>
            <property name="suffix">
             <string> [deg]</string>
            </property>
            <property name="value">
             <double>5.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBox_minSlewDistance">
            <property name="text">
             <string>min slew distance</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBox_maxSlewDistance">
            <property name="text">
             <string>max slew distance</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_minSlewDistance">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>minimum required slew distance to next source in degrees</string>
            </property>
            <property name="statusTip">
             <string>minimum required slew distance to next source in degrees</string>
            </property>
            <property name="suffix">
             <string> [deg]</string>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>5.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_maxSlewDistance">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>maximum allowed slew distance to next source in degrees</string>
            </property>
            <property name="statusTip">
             <string>maximum allowed slew distance to next source in degrees</string>
            </property>
            <property name="suffix">
             <string> [deg]</string>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>5.000000000000000</double>
            </property>
            <property name="value">
             <double>175.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QCheckBox" name="checkBox_maxNumberOfScans">
            <property name="text">
             <string>max number of scans</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="spinBox_maxNumberOfScans">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>maximum number of scans for this station</string>
            </property>
            <property name="statusTip">
             <string>maximum number of scans for this station</string>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>9999</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox_scanTime">
         <property name="title">
          <string>scan time</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>max scan time</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBox_maxScanTime">
              <property name="toolTip">
               <string>maximum allowed scan duration for this station in seconds</string>
              </property>
              <property name="statusTip">
               <string>maximum allowed scan duration for this station in seconds</string>
              </property>
              <property name="suffix">
               <string> [s]</string>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
              <property name="singleStep">
               <number>60</number>
              </property>
              <property name="value">
               <number>600</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>min scan time</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBox_minScanTime">
              <property name="toolTip">
               <string>minimum required scan time for this station in seconds</string>
              </property>
              <property name="statusTip">
               <string>minimum required scan time for this station in seconds</string>
              </property>
              <property name="suffix">
               <string> [s]</string>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="value">
               <number>30</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget_SNR">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>120</height>
             </size>
            </property>
            <property name="toolTip">
             <string>minimum required signal to noise ratio</string>
            </property>
            <property name="statusTip">
             <string>minimum required signal to noise ratio</string>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <row>
             <property name="text">
              <string>X</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>S</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>SNR</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_ignoreSources">
         <property name="toolTip">
          <string>ignore sources for this station</string>
         </property>
         <property name="statusTip">
          <string>ignore sources for this station</string>
         </property>
         <property name="title">
          <string>ignore sources</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>available</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListView" name="listView_ignoreSources">
                <property name="toolTip">
                 <string>list of available sources</string>
                </property>
                <property name="statusTip">
                 <string>list of available sources</string>
                </property>
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Filter:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_filter">
                  <property name="toolTip">
                   <string>filter list of available sources</string>
                  </property>
                  <property name="statusTip">
                   <string>filter list of available sources</string>
                  </property>
                  <property name="inputMask">
                   <string/>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>selected</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="listWidget_selectedIgnoreSources">
                <property name="toolTip">
                 <string>list of sources which will not be observed with this station</string>
                </property>
                <property name="statusTip">
                 <string>list of sources which will not be observed with this station</string>
                </property>
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>parameter name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="toolTip">
        <string>parameter name</string>
       </property>
       <property name="statusTip">
        <string>parameter name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="myresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>stationParametersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>614</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>571</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_weight</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBox_weight</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_maxSlewTime</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_maxSlewTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_maxWaitTime</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_maxWaitTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_minElevation</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBox_minElevation</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_minSlewDistance</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBox_minSlewDistance</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_maxSlewDistance</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBox_maxSlewDistance</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_maxNumberOfScans</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_maxNumberOfScans</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
